examples/embedded-sso/client/src/App.tsx

```diff
@@ -63,7 +62,7 @@ export default function App() {
     const [apiEnvironment, setApiEnvironment] = useState('sandbox');
     const [requesting, setRequesting] = useState(true);
 
-    let defaultIframeSource = `https://${companyCode}.swa.${apiEnvironment}.streem.cloud/logout-success`;
+    let defaultIframeSource = `https://${companyCode}.swaga.${apiEnvironment}.streem.cloud/logout-success`;
```

Comment:
❓ Do you know why this originally set the default path for SWA to `/logout-success`? There isn't an equivalent path in EV2 (it `404`s). There's `/logout-success.html` but that doesn't have a UI component (it's only for deleting the `fusionauth_refresh_token` post-logout).

Not exactly sure, but I think the best path here would be `/` or maybe `/login`.

Reply:
It was just a holding page until you generated the token.  If you go to `/`, it will redirect to login, which might be confusing.  Probably using `about:blank` would be just fine.

Reply:
Oh good catch! I had originally updated this to `/login` but that change got lost when I reset the branch to dump those changes I made to the "Load iframe" button. 

Will update.